classes/author.rb

```diff
@@ -10,4 +11,9 @@ def initialize(firstname, lastname)
     @lastname = lastname
     @items = []
   end
+
+  def add_item(item)
+    item.author = self
+    @items << item unless @items.include?(item)
```

Comment:
**Kindly switch these two lines**. _As Ruby will execute line by line it is always better to ensure your code follows order._

Reply:
Hello @Yazino12 , thanks for the review, kindly check the update code.